doozer/doozerlib/metadata.py

```diff
@@ -487,24 +498,6 @@ def latest_build_list(pattern_suffix):
                 # "assembly.howdy'.
                 refined = [b for b in builds if b['nvr'].endswith(pattern_suffix) or f'{pattern_suffix}.' in b['nvr']]
 
-                # .el? was added to images well after assemblies were established. It allows us to build multiple
```

Comment:
this doesn't apply now since we are specifically searching by elX

Reply:
This means we can no  longer assemble & fix older releases that didn't follow this convention. We should be prepared to run with an older commit of doozer if this occurs.